style.css

```diff
@@ -0,0 +1,245 @@
+*{
+	box-sizing: border-box; 
+}
+html{
+	font-size: 20px; 
+}
+body {
+	background-color:#771668;
+	width: 100%;
+	margin-top:80px;
+	margin-left:0px;
+	margin-right:0px;
+	margin-bottom:0px;
+}
+
+header{
+	position: fixed;
+	top:0px;
+	left:0px;
+	background-color: #2d094f;
+	width:100%;
+	padding: 10px;
+	border:3px solid black;
+	z-index:2;
+}
+
+.nav {
+	font-size: 1.5rem;
+	color: white;
+	display: flex;
+	gap:40px;
+	text-decoration: none;
+			
+}
+section.intro{
+		display: flex;
+		align-items: center;
+		flex-direction: column;
+		width: 100%;
+		margin-bottom:250px;
+		gap: 18px;			
+}
+
+p.intro{
+	color: white;
+	font-size: 6rem;
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap;
+	margin:0px;
+	align-items: center;
+	gap:0px;
+}
+
+p.little-intro{
+	color: white;
+	font-size: 2rem;
+	display: flex;
+	justify-content: center;
+}
+
+div.about-catcher{
+    width: 100%;
+ 	padding: 10px;
+    background-color: #3C0051;
+}
+
+section.about{
+	    width: 100%;
+	    height: 800px;
+	    background-color: #3C0051;
+	    margin-top: 30px;
+	    display: flex;
+	    flex-wrap: wrap;
+	    flex-direction: column;
+
+}
+
+.about-img{
+	width:600px;
+	height:600px;
+	object-fit:contain;	
+	margin-right: 10px;
+}
+
+h2.about{
+	margin-left:15px;
+	color: white;
+	font-size: 2rem;
+	text-decoration: underline;	
+}
+
+p.about{
+	width:60%;
+	color: white;
+	font-size: 1.5rem;
+	margin-left:15px;
+}
+
+div.portfolio-catcher {
+    width: 100%;
+ 	padding: 10px;
+    background-color: #453952;
+	margin-top:50px;
+}
+
+section.portfolio-intro { width: 100%;
+    height: 150px;
+  	background-color:#453952;
+	color: white;	
+}
+
+h2.portfolio-intro {
+	margin-left:15px;
+	text-decoration: underline;
+}
+
+p.portfolio-intro {
+	margin-left:15px;
+	color: white;
+	font-size: 1rem;
+}
+
+section.portfolio {
+	display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+	color: white;
+}
+
+
+div.portfolio-piece1 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece2 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
+
+div.portfolio-piece3 {
+	background-color: #3e2182;
+	margin-top: 30px;
+	border: 5px solid black;
+	width: 70%;
+	height: 600px;
+	}
```

Comment:
here it is self evident why they should all have 1 class - they all have the same styling rules

also, again, instead of using 70% width, you should use 100% width and give it padding.

Reply:
height 👀